### PR TITLE
Adding test for Active Directory Audit permissions

### PR DESCRIPTION
**Pull request by mcollera**

### CODE REVIEW

File: Tests/Unit/ActiveDirectoryAuditRuleEntry.Tests.ps1 [hunk 1]
```diff
@@ -0,0 +1,592 @@
...
+                            Principal = $Principal
+                            ForcePrincipal = $ForcePrincipal
+                            AccessControlEntry = [Microsoft.Management.Infrastructure.CimInstance[]]@($CimAccessControlEntry)
+                        }
+    }
+    Return $CimAccessControlList
+}
+
+#endregion
+
```
Comment (regedit32) [inline Tests/Unit/ActiveDirectoryAuditRuleEntry.Tests.ps1]: Can you check spacing throughout the file?  There's double and triple spaces in a lot of places

File: Tests/Unit/ActiveDirectoryAuditRuleEntry.Tests.ps1 [hunk 2]
```diff
@@ -0,0 +1,592 @@
...
+            DistinguishedName = "DC=PowerStig,DC=Local"
+
+            AccessControlList = $TempAcl
+
+        }
+
+        $GetResult = & "$($Global:DSCResourceName)\Get-TargetResource" @ContextParams
+
+        It 'Should return Ensure set as empty' {
+
```
Comment (regedit32) [inline Tests/Unit/ActiveDirectoryAuditRuleEntry.Tests.ps1]: It blocks don't need the extra spaces at beginning and end

File: Tests/Unit/ActiveDirectoryAuditRuleEntry.Tests.ps1 [hunk 3]
```diff
@@ -0,0 +1,592 @@
...
+Import-Module Pester -Force
+
+#endregion
+
+#region Helper Functions
+
+function New-AuditAccessControlList
+{
+<#
+    .SYNOPSIS
```
Comment (chasewilson) [inline Tests/Unit/ActiveDirectoryAuditRuleEntry.Tests.ps1]: The other helper functions are placed in the testhelper module. If you guys like it better this way, we can add them to other test scripts my only opinion is that we stay consistent.

Comment (regedit32) [reply Tests/Unit/ActiveDirectoryAuditRuleEntry.Tests.ps1]: I agree.  Additional functions should probably go in the helper

**Pull request merged**